### PR TITLE
Feature/emoji

### PR DESCRIPTION
**Pull request by processprocess**
Hello! SSS™

Added the visual sugar and emoji candy to our app! 
As well as navigation buttons, getting rid of some filler text and deleted commented out code.

Please give it a look and let me know of the +1 +2!

### CODE REVIEW

**Comment by Naser2** [comment]
Love it ProcessCss!! Great job!

File: client/components/App.jsx [hunk 1]
```diff
@@ -22,47 +21,67 @@ class App extends Component {
...
+    }
+    for (let i = 0; i < numHearts; i++) {
+    let randomEmoji = emojiArray[Math.floor(Math.random() * emojiArray.length) + 0  ]
+    $emojis = $('<h1>').addClass('emojis').html(randomEmoji);
+    $body.append($emojis);
+    $emojis.animate({
+      top: Math.floor(Math.random() * 200) - 50 + '%',
+      left: Math.floor(Math.random() * 200) - 50 + '%',
+      opacity: 0,
+      }, 1500, 'linear');
```
Comment (echeng924) [inline client/components/App.jsx:63]: Indentation for styling!

File: client/components/App.jsx [hunk 2]
```diff
@@ -22,47 +21,67 @@ class App extends Component {
...
       <div>
-        <div>
+        <div className="modeButtonHolder">
           <TypeButton
-          name="Light"
+          name="😇 Light"
           value="light"
           questionType={this.state.questionType}
           onTypeButtonClick={this.handleTypeButtonClick}
           />
```
Comment (echeng924) [inline client/components/App.jsx:76]: Indentation styling for <TypeButton />, contents should have one more indent

File: client/components/App.jsx [hunk 3]
```diff
@@ -22,47 +21,67 @@ class App extends Component {
...
           <TypeButton
-          name="Dark"
+          name="😈 Dark"
           value="dark"
           questionType={this.state.questionType}
           onTypeButtonClick={this.handleTypeButtonClick}
           />
```
Comment (echeng924) [inline client/components/App.jsx:82]: Indent for styling :)

**Pull request merged**